src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -187,7 +187,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('pull_request_trigger_enabled', help="Indicates whether the source control pull request trigger is enabled. The trigger is disabled by default.", arg_type=get_three_state_flag())
         c.argument('schedule', help="Schedule for a timer trigger represented as a cron expression. An optional trigger name can be specified using `--schedule name:schedule` format. Multiples supported by passing --schedule multiple times.", action='append')
         c.argument('git_access_token', help="The access token used to access the source control provider.")
-        c.argument('branch', help="The source control branch name.")
+        c.argument('branch', deprecate_info=c.deprecate(hide=True), help="The source control branch name. Please specify your source branch in the context parameter e.g. https://github.com/Azure-Samples/acr-build-helloworld-node.git#mybranch")
```

Comment:
why do you set `hide=True`? It will completely hide this argument from users.

Reply:
User cannot see the help information anymore.

Reply:
because we're going to deprecate this parameter. The user should pass the branch and directory in the context.

Reply:
but you changed the help information. User won't see the new help information if you set `hide=True`

Reply:
We decided to remove it completely after I already modified the help info. I saw no need to revert the change, but I can revert if it's an issue.